src/core/Akka.TestKit/TestKitBase_AwaitAssert.cs

```diff
@@ -62,16 +62,19 @@ public async Task AwaitAssertAsync(Action assertion, TimeSpan? duration=null, Ti
                 }
                 catch(Exception)
                 {
-                    var stopped = Now + t;
-                    if (stopped >= stop)
```

Comment:
The problem here is that we don't get 1 last chance to test the assertion if the first attempt failed and took too long